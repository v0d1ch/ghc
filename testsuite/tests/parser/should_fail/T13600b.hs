module T13600b where

f4 :: Maybe [Int] -> IO Int
f4 !(Just _) = return (sum x)
