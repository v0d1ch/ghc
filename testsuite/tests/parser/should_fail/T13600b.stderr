
T13600b.hs:3:3: error:
    Did you forget to activate -XBangPatterns ?
    If not, try adding a space before the bang
    ! (Just x)

